torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -413,47 +450,101 @@ def __init__(self, op_name: str, scalars_attr=None, algorithm_attr=None):
+                    _bf16_fusion(u, _gelu_fusion_1, users=2) for u in call_user1
+                ],
+                UnaryAttr("hardswish"): [
+                    _bf16_fusion(u, _hardswish_fusion, users=2) for u in call_user1
+                ],
+                UnaryAttr("hardsigmoid"): [
+                    _bf16_fusion(u, _hardsigmoid_fusion, users=1) for u in call_user1
+                ],
+                UnaryAttr("swish"): [
+                    _bf16_fusion(u, _silu_fusion, users=2) for u in call_user1
+                ],
+            }
+            return replacement_unary_fusion_patterns
+
+        for replace_patterns, check_bf16 in zip(
+            [_fp32_fusion_patterns(), _bf16_fusion_patterns()], [False, True]
+        ):
+            for unary_attr, patterns in replace_patterns.items():
+                _register_unary_fusion_lowering(
+                    patterns[0], unary_attr, computation_ops[0], check_bf16
+                )
+                _register_unary_fusion_lowering(
+                    patterns[1], unary_attr, computation_ops[1], check_bf16
+                )
+                _register_unary_fusion_lowering(
+                    patterns[2], unary_attr, computation_ops[2], check_bf16
+                )
         _leaky_relu_patterns = [
-            _leaky_relu_fusion(user) for user in _computation_user_3
+            [_leaky_relu_fusion(u) for u in call_user3],
+            [_bf16_fusion(u, _leaky_relu_fusion, users=3) for u in call_user1],
+        ]
+        _hardtanh_patterns = [
+            [_hardtanh_fusion(user) for user in call_user1],
+            [_bf16_fusion(u, _hardtanh_fusion, users=1) for u in call_user1],
         ]
-        _hardtanh_patterns = [_hardtanh_fusion(user) for user in _computation_user_1]
-        for pattern, computation_op in zip(_leaky_relu_patterns, computation_ops):
-            _register_leaky_relu_fusion_lowering(pattern, computation_op)
-        for pattern, computation_op in zip(_hardtanh_patterns, computation_ops):
-            _register_hardtanh_fusion_lowering(pattern, computation_op)
+        for patterns, check_bf16 in zip(_leaky_relu_patterns, [False, True]):
+            for pattern, computation_op in zip(patterns, computation_ops):
+                _register_leaky_relu_fusion_lowering(
+                    pattern, computation_op, check_bf16
+                )
+        for patterns, check_bf16 in zip(_hardtanh_patterns, [False, True]):
+            for pattern, computation_op in zip(patterns, computation_ops):
+                _register_hardtanh_fusion_lowering(pattern, computation_op, check_bf16)
 
     def _register_inplace_fusion():
         binary_ops = [aten.add, ops.add]
         inplace_fusion_op = mkldnn._convolution_pointwise_.binary
         outplace_fusion_op = mkldnn._convolution_pointwise.binary
-        computation_call = _computation_user_1[0]
+        computation_call = _conv_call(users=1)
         computation_op = computation_ops[0]
```

Comment:
I feel it make things more readable if you rename `computation_call` to `conv_call` and `computation_op` to `conv_op`.

Reply:
changed.